server_instructions/server_instructions.md

```diff
@@ -1,9 +1,18 @@
 ## Creative Server
 
-This is a vanilla Minecraft server. This server can be joined from default Minecraft launcher running version 1.18.
+This is a vanilla Minecraft server. This server can be joined from default Minecraft launcher running version 1.18.2.
```

Comment:
Maybe specify that it's specifically a creative server, since in the `Survival Server` header you specified it's a survival server.